rest_api/rest_api/urls.py

```diff
@@ -1,7 +1,21 @@
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from api import views
 
+from rest_framework.routers import DefaultRouter
+
+router = DefaultRouter()
```

Comment:
user path instead of router

Reply:
path = DefaultRouter() use this got error 
router.register('mstudentapi', views.StudentModelViewSet,basename='student')
 path('', include(router.urls)),
Error - router is not defined

Reply:
Path means url pattern search in Google url pattern and set path in urls.py

Reply:
Sanje call ma samjve ne